apps/01PigLatin.js

```diff
@@ -7,11 +7,87 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // Concept Check
+  //
+  // Please explain why we are using String.prototype.toLowerCase() here.
+  // For more info:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+  //
+  // Your Answer: We are using String.prototype.toLowerCase() to neutralize the
+  // user input and make it easier for our functions to analyze.
+  //
+  word  = word.toLowerCase();
 
+  // Concept Check
+  //
+  // What is the significance of setting vowelIndex to -1?
+  // This might help:
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+  //
+  // Your Answer: Since an array starts with an index of 0, when we set the
+  // vowelIndex to -1, we are setting the index to start before the start of
+  // the array.
+  //
```

Comment:
Not quite. We are setting the vowelIndex to -1 because an index of -1 does not exist. 

This is telling the app a vowel hasn't been found it. We could have set it to any number that wouldn't exist (-2,-3,-4...etc)